dist/docker/debian/build_docker.sh

```diff
@@ -98,7 +95,7 @@ run apt-get -y upgrade
 run apt-get -y --no-install-suggests install dialog apt-utils
 run bash -ec "echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections"
 run bash -ec "rm -rf /etc/rsyslog.conf"
-run apt-get -y --no-install-suggests install hostname supervisor openjdk-11-jre-headless python2 python3 python3-yaml curl rsyslog sudo
+run apt-get -y --no-install-suggests install hostname supervisor python2 python3 python3-yaml curl rsyslog sudo
```

Comment:
who's still using python2? (unrelated to this issue of course)

Reply:
Based on https://github.com/scylladb/scylladb/commit/27e326652b092f401f8e4e35023333a32a68dae9 it's related, I can remove it now